docs/antora/modules/ROOT/pages/management/docker.adoc

```diff
@@ -0,0 +1,129 @@
+= Docker commands
+Mona Bärenfänger <mona@lightcurve.io>
+:description: Describes how to manage Lisk Service with Docker.
+:toc:
+:idseparator: -
+:idprefix:
+:experimental:
+:imagesdir: ../assets/images
+
+:url_config: configuration/docker.adoc
+
+== Start
+
+.Inside of the `lisk-service` root folder
+[source,bash]
+----
+make up
+----
+
+This will start Lisk Service with the default configuration, which in turn will connect to a Lisk Core Mainnet node.
+
+In case a different node or network shall be used, xref:{url_config}[configure] Lisk Service accordingly.
+
+== Stop
+
+.Inside of the `lisk-service` root folder
+[source,bash]
+----
+make down
+----
+
+This will stop Lisk Service.
+
+== Show status of Docker containers
+
+[source,bash]
+----
+docker ps
+----
+
+== Reset Lisk Service database
+
+To reset the database of Lisk Service, drop the respective MYSQL and Redis databases.
```

Comment:
As a general remark, please ensure that we state `MySQL` with a little `y` instead of `MYSQL` everywhere in the documentation.

Reply:
OK, good point..now done 👍